tests/features/secure-files.feature

```diff
@@ -0,0 +1,67 @@
+Feature: VermillhFeature: Vermillion is able to handle secure files
+
+
+	 Scenario: Consumer publishes along with a valid token
+                Given Vermillion is running
+                When The consumer publishes along with a valid token
+		Then The response status should be 201
+	
+	Scenario: Consumer publishes along with file and timeseries data
+                Given Vermillion is running
+                When The consumer publishes along with a file and timeseries data
+                Then The response status should be 400
+
+ 	Scenario: Consumer publishes along with invalid token
```

Comment:
What does 'along with' mean? Shouldn't it just be 'consumer publishes using an invalid token?' Am I missing something here?

Reply:
Rephrase to 'Consumer publishes a secure file with an invalid token'